WORKSPACE

```diff
@@ -10,6 +10,12 @@ git_repository(
     remote = "https://github.com/oferb/startupos-binaries",
 )
 
+git_repository(
+    name = "startupos_binaries",
+    commit = "e4ada206e86f94ef78176fa6e78e0d25919ecf7a",
+    remote = "https://github.com/vmax/startupos-binaries",
```

Comment:
Please use my repo instead.

Reply:
@oferb I'll be able to if you downgrade the grpc plugin version to `v0.12.0` -- otherwise build will fail